packages/esm-service-queues-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -15,7 +15,7 @@ import { useServiceMetricsCount } from './queue-metrics.resource';
 import { useVisitQueueEntries } from '../active-visits/active-visits-table.resource';
 import styles from './clinic-metrics.scss';
 import { useQueueLocations } from '../patient-search/hooks/useQueueLocations';
-import { useQueues } from '../helpers/useQueues';
+import { useQueues } from '../hooks/useQueues';
```

Comment:
Per above comment, this should not be changed.  We should leave the original helper hook in place for now and not (yet) aim to refactor existing components.